LondonBritishLibrary/orient/BLorient799.xml

```diff
@@ -35,23 +32,90 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 365</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
-                        
-                        
+
+<msItem xml:id="ms_i1">
+<locus from="5r"/>
+<title ref="LIT1309Didesq" xml:lang="la">Didascalia Apostolorum</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="54r"/>
+<title ref="LIT4336Ledata" xml:lang="gez">Lǝdata ʾabawina manakosāt</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i3">
+<locus from="55v"/>
+<title ref="LIT6164GoodWorks" xml:lang="en">The Book of Good Works, on the services of the church and the duties of the priest</title>
+<incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ ፈጣሪ፡ ሕያው፡ ተናጋሪ። ወበላዕሌሁ፡ ንትወከል። ወቦቱ፡ ኵሉ፡ ይትከሀል፡
+መጽሐፈ፡ ምግባራት፡ ሠናይት፡ ለተግሣጸ፡ ቤተ፡ ክርስቲያን፡ ዐበይት፡ ጽሙድ፡ በክቡር፡ ኅብራት፡ ለቃል፡ ወምግባር፡ ሥርዓታት።</incipit>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i4">
+<locus from="70v"/>
+<title xml:lang="la" ref="LIT4019Caeremoniale">Caeremoniale sacerdotum</title>
+<incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ <gap reason="omitted" resp="PRS10171Wright"></gap>
+ንጽሕፍ፡ ዘይደሉ፡ እምሥርዓተ፡ ቅኔ፡ መልእክት፡ ዘክህነት፡ ዘይትፈቀድ፡ በውስተ፡ ጸሎታት፡ ወቅዳስያት።</incipit>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i5">
+<locus from="73r"/>
+<title xml:lang="la" ref="LIT2332serata">Regulae de hierarchia ecclesiastica</title>
+<incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="omitted" resp="PRS10171Wright"></gap> ንጽሕፍ፡
+ሥርዓተ፡ ቤተ፡ ክርስቲያን፡ ወመዓርጋቲሃ፡ ዘአንበሩ፡ አበው፡ ቅዱሳን።</incipit>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i6">
+<locus from="78r"/>
+<title xml:lang="la" ref="LIT4021Quaestiones">Quaestiones et interrogationes patrum earumque solutio</title>
+<incipit>ታኃሥሥ፡ እለታት፡ ዘአበው፡ ፍካሬሆሙ።</incipit>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i7">
+<locus from="81r"/>
+<title xml:lang="en" ref="LIT4018Mashafa">Faws manfasāwi, or Epitome of canon law</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i8">
+<locus from="89r"/>
+<title ref="LIT1395Fethan" xml:lang="gez">Fǝtḥa nagaśt</title>
+<incipit xml:lang="gez">በስመ፡ አብ፡<gap reason="omitted" resp="PRS10171Wright"></gap>ንዌጥን፡ በረድኤተ፡
+እ<hi rend="ligature">ግዚ</hi>እነ፡ ወስነ፡ ሠርሖቱ፡ በጸሒፈ፡ መቅድመ፡ ጉባኤ፡ ዘአስተጋብእዎ፡ ሊቃውንት፡ ተወጥነት፡
+ዛቲ፡ መጽሐፍ፡ በመንግሥተ፡ ንጉሥነ፡ <persName ref="PRS9152TaklaHa">ተክለ፡ ሃይማኖት፡</persName> አመ፡ ነግሠ፡ በ፫አውራኅ፡ <gap reason="omitted" resp="PRS10171Wright"/>
+በረከቶሙ፡ ትኩን፡ ምስለ፡ <gap reason="omitted" resp="PRS10171Wright"/> </incipit>
+<note>The text of the first part begins on f. 93v, of the second on f. 144r.</note>
```

Comment:
Please, encode the folia with locus

Reply:
Note, there is also information on the parts of the text, at least where they start. It would be better to encode this in the subdued ms_items